data/yam/agama/auto/lib/scripts_post.libsonnet

```diff
@@ -6,5 +6,13 @@
       #!/usr/bin/env bash
       echo 'PermitRootLogin yes' > /etc/ssh/sshd_config.d/root.conf
     |||
+  },
+  enable_gdm: {
```

Comment:
Not every test need to enable gnome, so need add condition to control the run for 'enable_gdm' in template.libsonnet.

Reply:
It was controled by `scripts_post=enable_root_login,enable_gdm`